src/pointtopoint.jl

```diff
@@ -188,9 +188,9 @@ of communicator `comm` using with the message tag `tag`
 
 Returns the commication `Request` for the nonblocking send.
 """
-function Isend(buf::MPIBuffertype{T}, count::Integer,
-               dest::Integer, tag::Integer, comm::Comm) where T
-    Isend(buf, count, mpitype(T), dest, tag, comm)
+function Isend(buf, count::Integer,
+               dest::Integer, tag::Integer, comm::Comm)
+    Isend(buf, count, mpitype(Base.eltype(buf)), dest, tag, comm)
```

Comment:
Note on test failure on Julia nightly: On commits https://github.com/JuliaParallel/MPI.jl/pull/286/commits/d452f85019746ac49e69a7b9e2e15972327e8b8e and https://github.com/JuliaParallel/MPI.jl/pull/286/commits/c0eb592a2b3f1dc28426b8ddb8d579c0a38c92d7, the test `test/test_cman_mpi.jl` failed on Julia nightly build only. The reason seemed to be this line, so I changed lines 205 and 219  to avoid this line altogether on `cman.jl`. What should be done with this line?